Simplified/SignInLogic/NYPLSignInBusinessLogic+OAuth.swift

```diff
@@ -66,8 +66,17 @@ extension NYPLSignInBusinessLogic {
   }
 
   //----------------------------------------------------------------------------
+  
+  // As per Apple documentatino, selector for NSNotification must have one and
+  // only one argument.
+  // See https://developer.apple.com/documentation/foundation/nsnotificationcenter/1415360-addobserver
+  // for more information.
+  @objc func handleRedirectURL(_ notification: Notification) {
+    self.handleRedirectURL(notification, completion: nil)
+  }
+  
   // this is used by both Clever and SAML authentication
-  @objc func handleRedirectURL(_ notification: Notification, completion: (_ error: Error?, _ errorTitle: String?, _ errorMessage: String?)->()) {
+  @objc func handleRedirectURL(_ notification: Notification, completion: ((_ error: Error?, _ errorTitle: String?, _ errorMessage: String?)->())?) {
```

Comment:
minor: I think `@objc`  can be removed from this one now?

Reply:
NYPLSAMLHelper is still using it so we can't remove it.